tests/PySys/docker_plugin/docker_plugin_remove/run.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+from environment_docker_plugin import DockerPlugin
+
+"""
+Validate docker plugin install
+
+Using `registry`[https://hub.docker.com/_/registry] as a test container
+
+"""
+
+
+class DockerPluginRemoveTest(DockerPlugin):
+
+    image_name = "registry"
+
+    def setup(self):
+        super().setup()
+        # Assert that an image with the given name is not present on the machine before test
+        self.assert_image_present(
+            self.image_name, negate=True, abortOnError=True)
+        # Assert that no containers using the given image name and version are running
+        self.assert_container_running(
+            self.image_name, negate=True, abortOnError=True)
```

Comment:
It seems that we can simplify this test.

Why do we check that these images/containers are not present/running? The very next step is to run these!
Worse, why do we abort here?

Reply:
The idea was to make sure that the image being used in test is not present on the machine before the test starts so that we can be sure that the env is clean(no stale containers or images) . Added `abortOnError` to skip the test if this precondition fails as there's no point in continuing. This `abortOnError` will only skip this test and not the whole test execution.